src/registrar/templates/includes/footer.html

```diff
@@ -87,19 +85,16 @@
             >
           </li>
           <li class="usa-identifier__required-links-item">
-            <a href="https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/site/cisagov/getgov-home/privacy-policy/" class="usa-identifier__required-link usa-link"
-              >Privacy policy</a
-            >
+ <a href="{% public_site_url 'privacy-policy' %}" class="usa-identifier__required-link usa-link">Privacy policy</a>
```

Comment:
**question (non-blocking)**: As above, this has removed the trailing slash. Is that intentional?